.github/workflows/release.yml

```diff
@@ -13,11 +13,17 @@ permissions:
 
 jobs:
 
+  test:
```

Comment:
will adding `if: startsWith(github.ref, 'refs/tags')` work? Otherwise we'll always runs the full matrix tests right?

Reply:
That's correct, I can skip the `test` for the `main` branch. That's already part of the default workflow with the scheduler.

Reply:
See https://github.com/elastic/apm-agent-python/pull/1998/commits/054de934800ac2dbc9e1d37c8c8793c9cf3e9dd6

A bit complicated, but that's how I managed to skip those tests for the releases without changing the `test.yaml`.

What do you think?

Reply:
LGTM